premis_event_service/views.py

```diff
@@ -597,26 +624,54 @@ def app_agent(request, identifier=None):
                 page = request.GET.get('page')
             else:
                 page = 1
-            atomFeed = makeObjectFeed(
-                paginator=Paginator(Agent.objects.all(), 20),
-                objectToXMLFunction=objectToAgentXML,
-                feedId=requestString[1:],
-                webRoot="http://%s" % request.META.get('HTTP_HOST'),
-                title="Agent Entry Feed",
-                idAttr="agent_identifier",
-                nameAttr="agent_name",
-                request=request,
-                page=page,
-            )
+            try:
+                atomFeed = makeObjectFeed(
+                    paginator=Paginator(Agent.objects.all(), 20),
+                    objectToXMLFunction=objectToAgentXML,
+                    feedId=requestString[1:],
+                    webRoot="http://%s" % request.META.get('HTTP_HOST'),
```

Comment:
The rationale described by Reed of single vs. double is what I believe the original coda programmer also went by. I seem to remember we started ending up with single and double used more arbitrarily/inconsistently than that and at one point decided to stick with single quotes unless the string itself contains single quotes and put this guideline in the local style guide.